slitlessutils/core/preprocess/crrej/drizzle.py

```diff
@@ -0,0 +1,70 @@
+from drizzlepac import astrodrizzle
+from pathlib import Path
+
+# Args taken from INS HSTaXe FullFrame Cookbooks
+COMMON_ARGS = {
+    'ir': {
+        'mdriztab': True,
+        'preserve': False,
+        'skysub': False,
+        'final_fillval': None
+    },
+    'uvis': {
+        'mdriztab': True,
+        'preserve': False,
+        'skysub': True,
+        'skymethod': 'match',
+        'final_fillval': None
+    },
+    'acs': {
+        'build': True,
+        'mdriztab': True,
+        'in_memory': False,
+        'skysub': False,
+        'driz_separate': False,
+        'median': False,
+        'blot': False,
+        'driz_cr': False,
+        'driz_sep_wcs': False,
+        'driz_combine': True,
+        'final_wcs': False
+    }
+}
+
+
+def drizzle(files, instrument=None, outdir=Path().absolute(), **kwargs):
```

Comment:
```suggestion
def drizzle(files, instrument, outdir=Path().absolute(), **kwargs):
```

`instrument` is not an optional input.